providers/openshift_redeploy_certificate.rb

```diff
@@ -25,18 +25,22 @@ def whyrun_supported?
   execute 'Backup master stuff' do
     command 'tar czvf master-backup-$(date +%s).tar.gz /etc/origin/master /var/www/html/master --ignore-failed-read && rm -rf /var/www/html/master'
     cwd '/etc/origin'
+    only_if '[ -a /etc/origin/master ]'
```

Comment:
Can use `node['cookbook-openshift3']['openshift_master_config_dir']` instead of hardcoding the directory